types/src/on_chain_config/consensus_config.rs

```diff
@@ -20,22 +20,30 @@ pub enum ConsensusAlgorithmConfig {
         quorum_store_enabled: bool,
         order_vote_enabled: bool,
     },
+    JolteonV3 {
+        main: ConsensusConfigV1,
+        quorum_store_enabled: bool,
+        order_vote_enabled: bool,
+        round_timeout_msg_enabled: bool,
```

Comment:
any reason this needs to be on-chain config? we should be able to support both modes and transition in two releases?